project/plugins.sbt

```diff
@@ -7,7 +7,7 @@ addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka"   % "0.12")
 addSbtPlugin("com.lightbend.sbt"  % "sbt-javaagent"      % "0.1.4")
 addSbtPlugin("com.thesamet"       % "sbt-protoc"         % "0.99.18")
 addSbtPlugin("com.typesafe.sbt"   % "sbt-git"            % "0.9.3")
-addSbtPlugin("com.typesafe.sbt"   % "sbt-twirl"          % "1.3.13")
+addSbtPlugin("com.typesafe.sbt"   % "sbt-twirl"          % "1.3.15")
```

Comment:
Why not 1.4.0?

Reply:
@scala-steward first proposes patch updates. It only proposes minor updates if the dependency is on the latest patch version.